server/image_pull.go

```diff
@@ -193,6 +205,10 @@ func (s *Server) pullImage(ctx context.Context, pullArgs *pullArguments) (string
 				log.Debugf(ctx, "Image %s already in store, skipping pull", img)
```

Comment:
⚠️ Now that policies can vary across tenants after #7016, a config digest comparison is not sufficient to establish trust; see elsewhere.

Reply:
This code path does not exist any more.